File: python/22.10/242.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        a, b = sorted(list(s)), sorted(list(t))
        if len(a) != len(b):
            return False
        for i in range(len(s)):
            if a[i] != b[i]:
                return False
        return True

File: typescript/23.08/242.ts
function isAnagram(s: string, t: string): boolean {
    const mem ={}
    if (s.length !== t.length) return false
    for (const x of s) {
        if (!mem[x]) {
            mem[x] = 0
        }
        mem[x]++
    }
    for (const x of t) {
        if (!mem[x]) {
            return false
        }
        mem[x]--
    }

    return true
};
